src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -528,6 +528,12 @@ impl WorkunitStore {
     }))
   }
 
+  pub fn setup_for_tests() -> WorkunitStore {
```

Comment:
Not sure if this needs docstring. I didn't know what to put in it.

Reply:
I think it need not have one .... iff you annotate this with `#[cfg(test)]`.